### ISSUE TITLE
$this->email->from name Error

### ISSUE THREAD

**Issue by isantolin** [ivantcholakov/codeigniter-phpmailer#6]
I am using CodeIgniter 2.2.1, When i tries to send mails Using GMail PHPMailer config profile and defines From Name in the controller, justs shows me "Root User" instead "Este es el nombre"

config/email.php
```
$config['mailfrom'] = 'xxxxxxxxx@gmail.com';

$config['useragent'] = 'PHPMailer';              // Mail engine switcher: 'CodeIgniter' or 'PHPMailer'
$config['protocol'] = 'smtp';                   // 'mail', 'sendmail', or 'smtp'
$config['mailpath'] = '/usr/sbin/sendmail';
$config['smtp_host'] = 'smtp.gmail.com';
$config['smtp_user'] = 'xxxxxxxxx@gmail.com';
$config['smtp_pass'] = 'xxxx';
$config['smtp_port'] = 465;
$config['smtp_timeout'] = 5;                        // (in seconds)
$config['smtp_crypto'] = 'ssl';                    // '' or 'tls' or 'ssl'
$config['smtp_debug'] = 0;                        // PHPMailer's SMTP debug info level: 0 = off, 1 = commands, 2 = commands and data, 3 = as 2 plus connection status, 4 = low level data output.
$config['wordwrap'] = true;
$config['wrapchars'] = 76;
$config['mailtype'] = 'html';                   // 'text' or 'html'
$config['charset'] = 'utf-8';
$config['validate'] = true;
$config['priority'] = 3;                        // 1, 2, 3, 4, 5
$config['crlf'] = "\n";                     // "\r\n" or "\n" or "\r"
$config['newline'] = "\n";                     // "\r\n" or "\n" or "\r"
$config['bcc_batch_mode'] = false;
$config['bcc_batch_size'] = 200;
$config['encoding'] = '8bit';                   // The body encoding. For CodeIgniter: '8bit' or '7bit'. For PHPMailer: '8bit', '7bit', 'binary', 'base64', or 'quoted-printable'.

```

and calls the code from a controller
```
$subject = 'Este es el Mail de Registracion';
 $name = "Este es el nombre";
$message = "Message";


                $this->email->set_mailtype("html");
                $this->email->from($this->config->item('mailfrom'), $name);
                $this->email->reply_to($this->config->item('mailfrom'));
                $this->email->to($this->input->post('Email'));
                $this->email->subject($subject);
                $this->email->message($message);
                $this->email->send();
```

**Comment by ivantcholakov** [comment]
Could you rephrase a little bit?

**Comment by isantolin** [comment]
```
$name = "Peter Lopez";
$this->email->from($this->config->item('mailfrom'), $name);
```
When i tries to Send a mail using $name parameter, send with "Root Name" instead "Peter Lopez"

**Comment by ivantcholakov** [comment]
Ok, I understand the issue. I will check it soon.

**Comment by isantolin** [comment]
@ivantcholakov Fails in Codeigniter 3.0rc3 too

**Comment by ivantcholakov** [comment]
I will test it now. First let me mention that if the setting
```php
$config['mailfrom'] = 'xxxxxxxxx@gmail.com';
```
is within the file config/email.php and if this file is read only by the Enail library (by default) then
```php
$this->config->item('mailfrom')
```
will return nothing. IMO you may move the setting $config['mailfrom'] within the file config/config.php

**Comment by ivantcholakov** [comment]
It work for me, I can see 'Test Name' in my message.

```php
    public function index() {

            $this->load->library('email');

            $subject = 'This is a test';
            $message = '<p>This message has been sent for testing purposes.</p>';

            // Get full html:
            $body =
'<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
    <title>'.htmlspecialchars($subject, ENT_QUOTES, $this->email->charset).'</title>
    <style type="text/css">
        body {
            font-family: Arial, Verdana, Helvetica, sans-serif;
            font-size: 16px;
        }
    </style>
</head>
<body>
'.$message.'
</body>
</html>';

            $result = $this->email
                ->from('ivantcholakov@gmail.com', 'Test Name')
                ->to('ivantcholakov@gmail.com')
                ->subject($subject)
                ->message($body)
                ->send();

            var_dump($result);
            echo '<br />';
            echo $this->email->print_debugger();

            exit;
    }
```

The only idea I have is you to check what value $this->config->item('mailfrom') contains.

**Comment by ivantcholakov** [comment]
This issue seems to me abandoned. I am closing it.
